.gitignore

```diff
@@ -137,3 +137,7 @@ sketch
 
 Client/.vscode/settings.json
 *.snap
+
+Client/package-lock.json
+
+Client/package.json
```

Comment:
we should not ignore package and package-lock files, npm uses package to check what packages are missing in the current environment, and package lock is there to ensure we got correct dependencies. We should be pushing package lock and package.json files